xbmc/cores/VideoPlayer/DVDInputStreams/InputStreamAddon.cpp

```diff
@@ -367,8 +367,10 @@ std::vector<CDemuxStream*> CInputStreamAddon::GetStreams() const
 
 CDemuxStream* CInputStreamAddon::GetStream(int streamId) const
 {
-  INPUTSTREAM_INFO stream = m_struct.toAddon->get_stream(&m_struct, streamId);
-  if (stream.m_streamType == INPUTSTREAM_INFO::TYPE_NONE)
+  INPUTSTREAM_INFO stream;
+  memset(&stream, 0, sizeof(INPUTSTREAM_INFO));
```

Comment:
Why not just

```c++
INPUTSTREAM_INFO stream{};
```
this should have the same effect, right?

Reply:
You're right, it's a holdover from the old days and no longer necessary.

Still partially overlook this one and will change it.